dspace_import.py

```diff
@@ -906,11 +1007,56 @@ def import_handle_with_url():
             json_p = {'handle': i['handle'], 'url': i['url']}
             try:
                 response = do_api_post('core/handles', None, json_p)
+                imported_handle+=1
             except:
                 log('POST response ' + response.url + ' failed. Status: ' + str(response.status_code))
 
     log("Handles with url were successfully imported!")
 
+def import_user_metadata():
+    """
+        Import data into database.
+        Mapped tables: user_metadata, license_resource_user_allowance
+        """
+    global eperson_id, bitstream_id, statistics, userRegistration_id
+    imported = 0
+    # read license_resource_user_allowance
+    # mapping transaction_id to mapping_id
+    user_allowance = dict()
+    json_a = read_json("license_resource_user_allowance.json")
+    if json_a:
+        for i in json_a:
+            user_allowance[i['transaction_id']] = i
+
+    # read license_resource_mapping
+    # mapping bitstream_id to mapping_id
+    resource_mapping = read_json('license_resource_mapping.json')
+    mappings = dict()
+    if resource_mapping:
+        for i in resource_mapping:
+            mappings[i['mapping_id']] = i['bitstream_id']
+
+    # read user_metadata
+    json_a = read_json("user_metadata.json")
+    if json_a:
```

Comment:
json_a if again

Reply:
There will be no information in the logs about a successful import if the JSON is empty. Do we want that information or not?